src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -338,8 +317,9 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         }
         CATCH_LOG()
 
-        *in = inPipePseudoConsoleSide.release();
-        *out = outPipePseudoConsoleSide.release();
+        _pipe = std::move(pipe.server);
+        *in = pipe.client.release();
```

Comment:
This creates a duplex pipe, so it doesn't really matter which side is the server. I gave the server to WT, because that loosely matches how console servers work, but it'd be the same result if we swapped the sides around and gave ConPTY the server side.